roles/network_plugin/calico/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think this part should be replaced by just removing these lines: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/network_plugin/meta/main.yml#L29-L31

Reply:
😀 It does achieve the same effect and is more concise, thank you very much, @cristicalin